{"name": "测试登录正常", "status": "broken", "statusDetails": {"message": "PermissionError: [WinError 32] 另一个程序正在使用此文件，进程无法访问。: 'D:\\\\B2B\\\\OutPuts\\\\log\\\\Web_Autotest_2019-11-20 1528.log'", "trace": "@pytest.fixture(scope=\"session\",autouse=True)  #默认调用\n    def session_action():\n        print(\"===== 会话开始，测试用例开始执行=====\")\n        #清除测试报告、截图目录\n>       remove_files_in_dir(contants.reports_log)\n\nconftest.py:24: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\ndir = 'D:\\\\B2B\\\\OutPuts\\\\log'\n\n    def remove_files_in_dir(dir):\n        files = os.listdir(dir)\n        for item in files:\n            c_path = os.path.join(dir,item)\n            if os.path.isdir(c_path):\n                remove_files_in_dir(c_path)\n            else:\n>               os.remove(c_path)\nE               PermissionError: [WinError 32] 另一个程序正在使用此文件，进程无法访问。: 'D:\\\\B2B\\\\OutPuts\\\\log\\\\Web_Autotest_2019-11-20 1528.log'\n\n..\\common\\delete_history.py:12: PermissionError"}, "description": "这是验证码登录的成功用例", "attachments": [{"name": "stdout", "source": "81a54737-7917-420a-ace5-6f3ba9060ab0-attachment.txt", "type": "text/plain"}], "start": 1574234922103, "stop": 1574234922103, "uuid": "ace61eb6-0863-40d5-b3d7-5d7d26369cef", "historyId": "a404df02f600ced5f37d055b093f39d6", "fullName": "Testcases.test_2buy.TestLogin#test_login", "labels": [{"name": "feature", "value": "每日用例-加购"}, {"name": "story", "value": "登录"}, {"name": "tag", "value": "demo"}, {"name": "parentSuite", "value": "Testcases"}, {"name": "suite", "value": "test_2buy"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "JS-CS-002"}, {"name": "thread", "value": "42280-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Testcases.test_2buy"}], "links": [{"type": "link", "url": "www.baidu.com", "name": "www.baidu.com"}]}